core/templates/pages/exploration-player-page/exploration-player-page.component.spec.ts

```diff
@@ -136,4 +144,37 @@ describe('Exploration player page', function() {
     expect(elementDescriptionProperty.attr('content')).toBe(
       exploration.objective);
   });
+
+  it('should initialize command executor service',
+    function() {
+      WindowWrapperMessageService.getLocationHref =
+      jasmine.createSpy('parentMessage spy').and.returnValue(
```

Comment:
Please follow the indentation conventions that are used elsewhere in the JS files:

```
WindowWrapperMessageService.getLocationHref = (
  jasmine.createSpy(...).and.returnValue(
    'https://...'
```

Also, this is a test file that's run in a local context, so you shouldn't be using https://www.oppia.org right?